### PR TITLE
Remove unused css in enableJS.jsp #2803

### LINKED ISSUE #2803: Remove unused css in enableJS.jsp
In enableJS.jsp, from lines 6 to 29, there are some unused css that have been commented out.

### PR DESCRIPTION
**Pull request by ruchirsharma1993**
Hello,
As per the specification , I have removed the unused css in enableJS.jsp. Handling the issue for the first time, kindly inform if there is any problem with the same.
Fixes #2803 
Thanks
 'firsttimer'

### CODE REVIEW

**Comment by LowWeiLin** [comment]
@ruchirsharma1993 I have just assigned the issue to you.
The pull request title, description, branch are in the wrong format for this pull request.

Do take a look at out development workflow here:
https://github.com/TEAMMATES/repo/blob/master/devdocs/process.md

**Comment by ruchirsharma1993** [comment]
Hello Sir,
Please evaluate the current pull request and instruct accordingly.
Thanks

File: src/main/webapp/enableJS.jsp [hunk 1]
```diff
@@ -1,32 +1,6 @@
 <noscript>
-<style>
-/* remove display of original page*/
-#frameBody, #frameTop, #frameBottom{display:none;}
+<style>
```
Comment (LowWeiLin) [inline src/main/webapp/enableJS.jsp]: You can remove this style tag too, since there is nothing left in it.

**Comment by LowWeiLin** [comment]
The branch name you are working on is still wrong. You should create a branch from master, name it "Issue2803", make fixes on that branch, and pull request from there. You may have to close this pull request and open another one.

**Pull request closed without merging**